### PR TITLE
Update App.php

### PR DESCRIPTION
**Pull request by geggleto**
Fix for #1270
Adding return to every code path

### CODE REVIEW

**Comment by silentworks** [comment]
Am I being picky by asking why not use a switch statement instead of if else if and else?

**Comment by geggleto** [comment]
@silentworks I really just hate switch statements, haha.

**Comment by silentworks** [comment]
I hate that you hate switch statements. ;-(

File: Slim/App.php [hunk 1]
```diff
@@ -351,13 +351,13 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
...
-        if ($routeInfo[0] === \FastRoute\Dispatcher::NOT_FOUND) {
-            $notFoundHandler = $this->container->get('notFoundHandler');
-            return $notFoundHandler($request, $response);
-        }
-        if ($routeInfo[0] === \FastRoute\Dispatcher::METHOD_NOT_ALLOWED) {
+        else if ($routeInfo[0] === \FastRoute\Dispatcher::METHOD_NOT_ALLOWED) {
             $notAllowedHandler = $this->container->get('notAllowedHandler');
             return $notAllowedHandler($request, $response, $routeInfo[1]);
+        }        
+        else {
```
Comment (akrabat) [inline Slim/App.php]: Don't need this `else`.

**Comment by codeguy** [comment]
Switch statements are cool if you have A LOT of options. Otherwise just use an if statement :)

**Pull request merged**